app/models/user.rb

```diff
@@ -26,4 +29,16 @@ def set_current_block(block)
   def reset_current_block
     update_attribute(:current_block_id, nil)
   end
-end
\ No newline at end of file
+
+  private
+
+  def locale_as_available
+    unless I18n.available_locales.include?(locale.to_sym)
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.